### PR TITLE
Remove properties passing down to the stream child components and use store inheritance

### PR DESCRIPTION
**Pull request by christophehenry**
- [x] Remove properties passing down to the stream child components and use store inheritance
- [x] I can haz tests?

### CODE REVIEW

File: socialhome/streams/app/components/StreamElement.vue [hunk 1]
```diff
@@ -41,31 +30,27 @@ import store from "streams/app/stores/applicationStore"
...
         deleteUrl() {
-            return Urls["content:delete"]({pk: this.id})
+            return Urls["content:delete"]({pk: this.contentId})
+        },
+        editedText() {
+            return this.content.edited ? `${this.content.humanizedTimestamp} (edited)` : this.content.humanizedTimestamp
+        },
+        isUserAuthor(){
+            return this.$store.state.applicationStore.currentBrowsingProfileId === this.content.author.guid
+        },
```
Comment (christophehenry) [inline socialhome/streams/app/components/StreamElement.vue]: Can probably be removed (and doesn't work anyway)

**Comment by christophehenry** [comment]
Ready for review.

File: socialhome/streams/app/components/AuthorBar.vue [hunk 2]
```diff
@@ -94,17 +93,17 @@ export default Vue.component("author-bar", {
                 console.error("Not logged in")
                 return
             }
-            this.$http.post(`/api/profiles/${this.currentBrowsingProfileId}/remove_follower/`, {guid: this.guid})
-                .then(() => { this.$data._isUserFollowingAuthor = false })
+            this.$http.post(`/api/profiles/${this.currentBrowsingProfileId}/remove_follower/`, {guid: this.author.guid})
+                .then(() => { this.author.isUserFollowingAuthor = false })
```
Comment (jaywink) [inline socialhome/streams/app/components/AuthorBar.vue:97]: This works and updates the store value even though `this.author` is a `computed` value?

File: socialhome/streams/app/components/StreamElement.vue [hunk 3]
```diff
@@ -1,25 +1,14 @@
...
-            :has-shared="hasShared"
-        >
+        <div v-html="content.htmlSafe"></div>
+        <author-bar v-if="showAuthorBar" :content-id="contentId" />
+        <reactions-bar :content-id="contentId">
             <div class="mt-1">
-                <a :href="contentUrl" :title="timestamp" class="unstyled-link">
-                    {{ humanizedTimestamp }}<span v-if="edited"> {{ editedText }}</span>
+                <a :href="content.contentUrl" :title="content.timestamp" class="unstyled-link">
+                    {{ editedText }}
```
Comment (jaywink) [inline socialhome/streams/app/components/StreamElement.vue]: `timestampText` would be better considering the refactoring 👍

**Review by jaywink — CHANGES_REQUESTED** [review]
One minor comment in addition to the question - but then I assume it does work because you've tested it ;)

File: socialhome/streams/app/components/AuthorBar.vue (hunk 2 above)
Comment (christophehenry) [reply socialhome/streams/app/components/AuthorBar.vue:97]: Yep :)

**Review by jaywink — APPROVED** [review]

**Comment by jaywink** [comment]
Thanks! <3<3

**Pull request merged**